packages/x-data-grid-pro/src/components/headerFiltering/GridHeaderFilterMenu.tsx

```diff
@@ -43,29 +43,13 @@ function GridHeaderFilterMenu({
     apiRef.current.hideHeaderFilterMenu();
   }, [apiRef]);
 
-  const handleListKeyDown = React.useCallback(
-    (event: React.KeyboardEvent) => {
-      if (event.key === 'Tab') {
-        event.preventDefault();
```

Comment:
I'm not sure why we were doing this, but pressing the tab key should behave normally regardless of whether there is a menu open or not IMO. Curious if anyone has opinions/context to the contrary.